MOBILE_API.xml

```diff
@@ -2881,6 +2968,7 @@
         <element name="REMOTE_CONTROL"/>
         <element name="APP_SERVICES" since="5.1"/>
         <element name="SEAT_LOCATION" since="6.0"/>
+	<element name="DISPLAYS" since="6.0"/>
```

Comment:
Please Fix Spacing

Reply:
fixed in a8b4b1930782409dc886831843c72e223a77bcaa